guides/screen-reader-differences.md

```diff
@@ -0,0 +1,126 @@
+In this guide:
+
+- [Screen reader overview](#Overview)
+  - [Commonalities](#Commonalities)
+  - [Differences](#Differences)
+      - [Interaction modes](#Interaction-modes)
+- [Lessons](#lessons)
+
+# Overview
+
+Screen reader software allows users to navigate webpages by having the text of the page read aloud to them linearly. Screen reader users can also interact with the elements on the page using special keyboard commands that are unique to each application. The top 3 screen readers according to the [2017 WebAIM Survey](https://webaim.org/projects/screenreadersurvey7/) are JAWS, NVDA, and VoiceOver:
+
+|  Screen Reader |  % of respondents who commonly use it | 
+|---|---|
+|  JAWS | 66.0%  |
+|  NVDA | 64.9%  |
+|  VoiceOver | 39.6%  |
+
+JAWS and NVDA are Windows applications. VoiceOver is available as a desktop Mac OS application and on iOS devices.
+
+
+## Commonalities
+
+It is important to keep in mind that screen readers "present content linearly to users, one item at a time. This contrasts with the way in which most people use visual interfaces....Users must progress through such systems in a step-wise manner. The insight that audio interfaces are linearized versions of web content is an important one that should guide web developers during the engineering and design process." [Source](https://webaim.org/techniques/screenreader/)
+
+Accessibility guidelines that target screen reader users will benefit users no matter the software they use. Some things to keep in mind include:
+
+
+- "You can't navigate through the headings of a web page if the page doesn't have headings." 
+- "You can't hear the meaning of a graphic unless it has alternative text."
+- "Non-descriptive link text such as 'click here' and 'more' offer little or no clues as to what will happen when the user selects them." [Source](https://webaim.org/articles/screenreader_testing/)
+
+"Following the principles of the Web Content Accessibility Guidelines (WCAG) or Section 508 guidelines will definitely help remove barriers, though they are not fool-proof. As a matter of fact, no method is fool-proof. You'll have the most success by paying attention to principles and guidelines and **testing your content**." (Emphasis mine) [Source](https://webaim.org/articles/screenreader_testing/)
```

Comment:
Should "emphasis mine" be "emphasis intentionally added" to remove ownership of the file?

Reply:
Totally minor +1 to @jimmynotjim's suggestion here